libraries/AP_HAL_ChibiOS/hwdef/ARKV6X-bdshot/hwdef-bl.data

```diff
@@ -0,0 +1 @@
+include ../ARKV6X/hwdef-bl.dat
```

Comment:
Instead of this file which requires you build a bootloader put this in your hwdef:
```
USE_BOOTLOADER_FROM_BOARD ARKV6X
```